types/mongoose-lean-virtuals/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for mongoose-lean-virtuals 0.5
+// Project: https://github.com/vkarpov15/mongoose-lean-virtuals
+// Definitions by: Isaac Herrera <https://github.com/isaacdecoded>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import { Schema } from 'mongoose';
+
+declare function mongooseLeanVirtuals(schema: Schema): void;
+
+declare namespace mongooseLeanVirtuals {
+}
+
+export default mongooseLeanVirtuals;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
Oh! You are right, so sorry! Change has been pushed. Thanks!